src/css/index.css

```
@@ -0,0 +1,101 @@
+/* stylelint-disable indentation */
+@import "./stars.css";
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  --main-text-color: #060b35;
+  --secondary-text-color: #616070;
+  --button-main-color: #00acdc;
+
+}
+
+.card {
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-top: 16px;
+  align-self: center;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  color: var(--main-text-color);
+  margin-top: 40px;
+  line-height: 18px;
+}
+
+.card__code {
+  color: var(--secondary-text-color);
+  line-height: 14px;
+  margin-top: 4px;
+}
+
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.card__review-text {
+  line-height: 14px;
+  color: var(--main-text-color) ;
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: 24px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary-text-color);
+}
+
+.card__price-amount {
+  color: var(--main-text-color);
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+}
+
+.card__button {
+  text-transform: uppercase;
+  font-size: 14px;
+  font-weight: 700;
+  color: #fff;
+  text-align: center;
+  padding: 12px 70px 12px 69px;
```

I believe it could be this easy (1px is not a big deal)
```suggestion
  padding: 12px 70px;
```